docs/Rules/MA0075.md

```diff
@@ -1,12 +1,28 @@
 # MA0075 - Do not use implicit culture-sensitive ToString
 
+This rule reports cases where an implicit call to `obj.ToString()` may differ based on the current culture:
+
 ````csharp
 _ = "abc" + -1; // non-compliant as the result depends on the current culture
 
 _ = "abc" + -1.ToString(CultureInfo.CurrentCulture);   // compliant
 _ = "abc" + -1.ToString(CultureInfo.InvariantCulture); // compliant
 ````
 
+A given type is considered to have a culture-sensitive `ToString()` implementation if it implements **System.IFormattable**.<br/>
+An exception is made for the following types:
```

Comment:
You can also specify that positive-numbers are excluded

Reply:
I mean the analyzer already skips positive integer numbers (or look alike positive).

https://github.com/meziantou/Meziantou.Analyzer/blob/main/src/Meziantou.Analyzer/Rules/DoNotUseImplicitCultureSensitiveToStringAnalyzer.cs#L200

You are right on the floating numbers. I think we should update the code to remove the special case for Single/Double/Decimal.

Reply:
> Perhaps there are even other possible issues with this. Could there, for example, be a locale that always shows a sign for positive numbers?

The code doesn't use the culture at all for positive numbers (for Int32 and other integer values). You can check the implementation here: https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/Int32.cs,106